expansions/pandemonium/src/main/java/ladysnake/pandemonium/common/entity/ability/WitherSkullAbility.java

```diff
@@ -0,0 +1,137 @@
+package ladysnake.pandemonium.common.entity.ability;
+
+import ladysnake.requiem.api.v1.entity.ability.DirectAbility;
+import ladysnake.requiem.api.v1.entity.ability.IndirectAbility;
+import ladysnake.requiem.common.entity.ability.AbilityBase;
+import ladysnake.requiem.common.entity.ability.DirectAbilityBase;
+import ladysnake.requiem.common.entity.ability.IndirectAbilityBase;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.boss.WitherEntity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.entity.projectile.WitherSkullEntity;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.util.math.Vec3d;
+
+import java.util.Random;
+
+public class WitherSkullAbility extends AbilityBase<WitherEntity> {
+    private static final Random RANDOM = new Random();
+
+    public WitherSkullAbility(WitherEntity owner, int cooldownTime) {
+        super(owner, cooldownTime);
+    }
+
+    private static double getHeadX(WitherEntity owner, int headIndex) {
+        if (headIndex <= 0) {
+            return owner.getX();
+        } else {
+            float f = (owner.bodyYaw + (float)(180 * (headIndex - 1))) * 0.017453292F;
+            float g = MathHelper.cos(f);
+            return owner.getX() + (double)g * 1.3D;
+        }
+    }
+
+    private static double getHeadY(WitherEntity owner, int headIndex) {
+        return headIndex <= 0 ? owner.getY() + 3.0D : owner.getY() + 2.2D;
+    }
+
+    private static double getHeadZ(WitherEntity owner, int headIndex) {
+        if (headIndex <= 0) {
+            return owner.getZ();
+        } else {
+            float f = (owner.bodyYaw + (float)(180 * (headIndex - 1))) * 0.017453292F;
+            float g = MathHelper.sin(f);
+            return owner.getZ() + (double)g * 1.3D;
+        }
+    }
+
+    private static WitherSkullEntity summonSkullWithTarget(WitherEntity owner, double j, double k, double l) {
```

Comment:
That method should be `protected` instead of `private static`. Also could you name j, k, and l ?